Survey/Survey/Sources/Supports/Helper/Typealiases/Typealiases.swift

```diff
@@ -9,3 +9,6 @@
 import Foundation
 
 typealias ImageAssets = R.image
+typealias ColorAssets = R.color
+typealias FontAssets = R.font
```

Comment:
Worpt code has a great example for this using of R.font. What do you think?

```
extension UIFont {

    static func light(ofSize size: FontSize) -> UIFont {
        FontFamily.prompt.light(ofSize: size)
    }

    static func regular(ofSize size: FontSize) -> UIFont {
        FontFamily.prompt.regular(ofSize: size)
    }

    static func semiBold(ofSize size: FontSize) -> UIFont {
        FontFamily.prompt.semiBold(ofSize: size)
    }

    static func medium(ofSize size: FontSize) -> UIFont {
        FontFamily.prompt.medium(ofSize: size)
    }
}

extension UIFont {

    enum FontSize: CGFloat {

        /// 11px
        case xxxSmall = 11.0

        /// 12px
        case xxSmall = 12.0

        /// 13px
        case xSmall = 13.0

        /// 14px
        case small = 14.0

        /// 16px
        case regular = 16.0

        /// 18px
        case medium = 18.0

        /// 20px
        case large = 20.0

        /// 24px
        case headingLarge = 24.0

        /// 30px
        case xLarge = 30.0

        /// 32px
        case xxLarge = 32.0
    }

    enum FontFamily {

        case prompt

        func light(ofSize size: FontSize) -> UIFont {
            Font.promptLight(size: size.rawValue) ?? .systemFont(ofSize: size.rawValue)
        }

        func regular(ofSize size: FontSize) -> UIFont {
            Font.promptRegular(size: size.rawValue) ?? .systemFont(ofSize: size.rawValue)
        }

        func medium(ofSize size: FontSize) -> UIFont {
            Font.promptMedium(size: size.rawValue) ?? .systemFont(ofSize: size.rawValue, weight: .medium)
        }

        func semiBold(ofSize size: FontSize) -> UIFont {
            Font.promptSemiBold(size: size.rawValue) ?? .systemFont(ofSize: size.rawValue, weight: .semibold)
        }
    }
}

```

Reply:
update in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/41/commits/f10d87bdfe1267e19fe24bb94768bd66860444a7)